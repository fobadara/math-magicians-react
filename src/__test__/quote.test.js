import { render, cleanup } from '@testing-library/react';
import Quotes from '../components/quote';

afterEach(cleanup);

test('should create a snapshot', () => {
  const { asFragment } = render(<Quotes />);
  expect(asFragment(<Quotes />)).toMatchSnapshot();
});
