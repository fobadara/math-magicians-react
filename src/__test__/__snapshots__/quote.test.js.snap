// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create a snapshot 1`] = `
<DocumentFragment>
  <div
    class="d-flex align-items-center justify-content-center height"
  >
    <blockquote
      class="blockquote text-right"
    >
      <p>
        Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding
      </p>
      <footer
        class="blockquote-footer text-white"
      >
        <cite
          title="Source Title"
        >
          William Paul Thurston
        </cite>
      </footer>
    </blockquote>
  </div>
</DocumentFragment>
`;
